onnx/test/symbolic_shape_test.py

```diff
@@ -60,9 +60,9 @@ def _get_shape_from_name(
         for v in inputs + outputs + valueinfos:
             if v.name == name:
                 if v.type.HasField("tensor_type"):
-                    return v.type.tensor_type.shape
-                elif v.type.HasField("sparse_tensor_type"):
-                    return v.type.sparse_tensor_type.shape
+                    return v.type.tensor_type.shape  # type: ignore
+                if v.type.HasField("sparse_tensor_type"):
+                    return v.type.sparse_tensor_type.shape  # type: ignore
```

Comment:
May I understand which kind of error does it ignore?

Reply:
I don't think mypy is seeing `v.type.sparse_tensor_type.shape` as a `TensorShapeProto`. I did not check why since it is coming from protobuf. I suggest fixing it in another PR. Right now, it explicitely ignores what was implicitely ignored.